.storybook/utils/zeroReports.js

```diff
@@ -0,0 +1,19 @@
+export function zeroAnalyticsReport( report ) {
```

Comment:
(I've spent too long in the UK to recall if using "zero" as a verb is more of a Britishism.)

It looks like we're using "zero" as a verb here. I know that's _technically_ correct… but I find it kinda hard to read and I imagine especially in a programming context it could be confusing, especially for a non-native English speaker. Especially where we have a lot of "hasZeroData" type functions around, we're always using it a noun. "Clear" or even "zero out" seem clearer verbs here, but mostly as it's not a heavily-used, test function I'd go with `replaceValuesInAnalyticsReportWithZeroData` or something sufficiently verbose 😅 

Anyway, can we name this something where the verb is a bit more explicit? 😅

Reply:
I see your point here, happy to change this. Curious about function name lengths, I ran a bash one-liner to search for long camel-casey identifiers in `assets/js`:

`egrep -no '[a-zA-Z]{20,}' $( find assets/js -name "*.js" ) | cut -d : -f 3 | awk '{ print length, $0 }' | sort -ns | uniq`

I found that your suggestion of `replaceValuesInAnalyticsReportWithZeroData`, at 42 chars, would not be the longest function name, but in fact joint 5th place, tied with `getInsufficientPermissionsErrorDescription`. :)

```
42 getInsufficientPermissionsErrorDescription
43 receiveAccountsPropertiesProfilesCompletion
44 GatheringDataEntityURLZeroDataStatesDisabled
44 calculateDifferenceBetweenChartValuesTestSet
46 isAwaitingAccountsPropertiesProfilesCompletion
```

As you've pointed out, it's only going to be used in a handful of stories, so the extra word count doesn't weigh too heavily against it. I've made the change to `replaceValuesInAnalyticsReportWithZeroData`.